app-antivirus/clamtk/clamtk-5.25.ebuild

```diff
@@ -0,0 +1,170 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="af ar ast az bg bs ca cs da de el_GR en_AU en_CA en_GB eo es eu fi fo fr ga gl he hr hu id it ja km ko lo lt lv mr ms nb nl nl_BE nn pa pl pt pt_BR ro ru si sk sl sr@latin sv ta te th tr ug uk uz zh_CN zh_TW"
+PLOCALE_BACKUP="en_GB"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit desktop l10n perl-functions python-single-r1 xdg-utils
+
+MY_PV_CINNAMON="0.04"
+MY_PV_GNOME="0.02"
+MY_PV_KDE="0.18"
+MY_PV_XFCE="0.06"
+
+DESCRIPTION="A graphical front-end for ClamAV"
+HOMEPAGE="https://dave-theunsub.github.io/clamtk/"
+SRC_URI="https://bitbucket.org/davem_/clamtk/downloads/${P}.tar.xz
+	cinnamon? ( https://bitbucket.org/davem_/nemo-sendto-${PN}/downloads/nemo-sendto-${PN}-${MY_PV_CINNAMON}.tar.xz )
+	gnome? ( https://bitbucket.org/davem_/${PN}-gnome/downloads/${PN}-gnome-${MY_PV_GNOME}.tar.xz )
+	kde? ( https://bitbucket.org/davem_/${PN}-kde/downloads/${PN}-kde-${MY_PV_KDE}.tar.xz )
+	xfce? ( https://bitbucket.org/davem_/thunar-sendto-${PN}/downloads/thunar-sendto-${PN}-${MY_PV_XFCE}.tar.xz )"
+
+LICENSE="|| ( Artistic GPL-1+ )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cinnamon gnome kde +nls xfce"
+REQUIRED_USE="gnome? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="app-antivirus/clamav
+	dev-perl/File-chdir
+	dev-perl/Gtk2
+	dev-perl/JSON
+	dev-perl/LWP-Protocol-https
+	dev-perl/LWP-UserAgent-Cached
+	dev-perl/Locale-gettext
+	dev-perl/Text-CSV
+	dev-perl/glib-perl
+	dev-perl/libwww-perl
+	virtual/perl-Digest-MD5
+	virtual/perl-Digest-SHA
+	virtual/perl-Encode
+	virtual/perl-MIME-Base64
+	virtual/perl-Time-Piece
+	cinnamon? ( gnome-extra/nemo )
+	gnome? ( dev-python/nautilus-python[${PYTHON_USEDEP}]
+		gnome-base/nautilus )
+	kde? ( kde-apps/dolphin:5 )
```

Comment:
are those deps really necessary? why would you need a dep on dolphin to simply put a file into `/usr/share/kservices5/ServiceMenus/` 
same applies to cinnamon and xfce. ditch the deps.

gnome is different since the script imports python modules.
btw, I think it's better to rename gnome useflag to `nautilus`, look the tree, it's pretty common and makes sense.
and that makes me think it may be better to rename other flags (except kde, since it's desktop-wide plugin) to match file managers.

Reply:
I thought about it, I tend to drop it. When I look into the eBuild of `p7zip`, it's also not depending on them. So I renamed all use flags (kde,nautilus,nemo,thunar) and dropped those deps.